.github/workflows/docs.yml

```diff
@@ -0,0 +1,30 @@
+name: Publish API documentation
+
+on:
+  workflow_dispatch:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+*
+
+permissions: {}
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
```

Comment:
Not a blocker, but: Do you mind if we pin this to `ubuntu-22` or a specific version? I just don't want this dying because GitHub moved the dependencies and didn't tell us

Reply:
[2fd3b96](https://github.com/electron/notarize/pull/217/commits/2fd3b96666077463b15b1eb966eeb51677200fc7)